src/main/java/io/openapitools/swagger/GenerateMojo.java

```diff
@@ -127,6 +136,49 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         });
     }
 
+    private Application resolveApplication(JaxRSScanner reflectiveScanner) {
```

Comment:
I am wondering whether this would fit more naturally into the `JaxRSScanner` class instead? Of course it would then be necessary to pass down possible application class given to the plugin.

Reply:
Seems like a reasonable idea. I will have a look.